pkg/cluster/clusterutil/cluster.go

```diff
@@ -32,27 +31,16 @@ type Repository interface {
 }
 
 type repositoryT struct {
-	repo *repository.V1Repository
+	repo repository.Repository
 }
 
 // NewRepository returns repository
 func NewRepository(os, arch string) (Repository, error) {
-	profile := localdata.InitProfile()
-	mirror := repository.NewMirror(environment.Mirror(), repository.MirrorOptions{
-		Progress: repository.DisableProgress{},
```

Comment:
This is one different change, DisableProgress behavior is introduced  by https://github.com/pingcap/tiup/commit/5b045ab358a13c99845b4743d1cf6f8164770b3b